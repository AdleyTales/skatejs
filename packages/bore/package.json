{
  "author": "Trey Shugart <treshugart@gmail.com> (https://github.com/treshugart)",
  "browser": "dist/esm/index.js",
  "dependencies": {
    "@skatejs/val": "0.4.0",
    "skatejs-dom-diff": "1.0.0"
  },
  "description": "Enzyme-like testing for the DOM.",
  "devDependencies": {
    "@skatejs/ssr": "0.19.1",
    "@skatejs/val": "0.4.0"
  },
  "files": [
    "dist/cjs",
    "dist/esm"
  ],
  "keywords": [
    "web",
    "components",
    "custom",
    "elements",
    "shadow",
    "dom"
  ],
  "license": "MIT",
  "main": "dist/cjs/index.js",
  "module": "dist/esm/index.js",
  "name": "@skatejs/bore",
  "repository": "skatejs/skatejs",
  "scripts": {
    "test": "jest"
  },
  "version": "4.0.3",
  "bugs": "https://github.com/skatejs/skatejs/issues",
  "homepage": "https://skatejs.netlify.com"
}