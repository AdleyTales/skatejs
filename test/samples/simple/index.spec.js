import { mount } from '@skatejs/bore';
import './';

describe('samples/simple', () => {
  it('renders what we expect', () => {
    const el = document.createElement('hello-simple');
    return mount(el).wait(e => {
      expect(e.shadowRoot.innerHTML).toBe('Hello, <slot></slot>!');
    });
  });
});
