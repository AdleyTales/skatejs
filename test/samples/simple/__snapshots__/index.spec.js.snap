// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`samples/simple renders what we expect 1`] = `"Hello, <slot></slot>!"`;
